tool/site/client/src/components/send/resultSummaries.tsx

```diff
@@ -17,7 +17,7 @@ export const ResultSummaries: React.FC<ResultSummariesProps> = ({sendResult}) =>
       <Label isPageHeading>Send Results</Label>
       <ButtonList>
         <Button onClick={() => copyPrescriptionIds(sendResult)}>Copy Prescription IDs</Button>
-        {sendResult.results.every(s => s.success !== "unknown")
+        {sendResult.results.every(s => s.success !== "unknown" && !s.success)
```

Comment:
does this new check invalidate the old one?

Reply:
no as the requests are batched deltas so the success status of unknown is what they all start as and overtime they all switch to true|false, if any are false we want to show the exception report

Reply:
perfect mate makes sense

Reply:
just the success for the moment, can pick up a negative test-case in future pr